eng/pipelines/templates/Apex_Tests_On_Windows_Daily.yml

```diff
@@ -0,0 +1,233 @@
+
+              $(DartLab.Path)\Scripts\VisualStudio\Build\Get-BaselineBuildCommitIDs.ps1 -BuildJson $baselineBuildsFile -OutVariableName "BaselineBuildCommitIds"
+              } catch {
+                Write-Host "Unable to get Baseline build commit ids: $_"
+              }
+        - task: PowerShell@2
+          name: SetVisualStudioBaseBuildID
+          displayName: Set 'VisualStudio.BaseBuild.ID' 
+          retryCountOnTaskFailure: 3
+          condition: ne(variables['BaselineBuildCommitIds'], '')
+          continueOnError: true
+          inputs:
+            filePath: $(DartLab.Path)\Scripts\VisualStudio\Build\Get-BaselineBuildID.ps1
+            arguments: -OAuthAccessToken (ConvertTo-SecureString '$(System.AccessToken)' -AsPlainText -Force) -InstanceURL '$(System.CollectionUri)' -RepositoryName 'VS' -ProjectName '$(System.TeamProject)' -CommitIDs $(BaselineBuildCommitIds) -BuildDefinitionIDs 10289 -OutVariableName 'VisualStudio.BaseBuild.ID'
+
+        - task: PowerShell@2
+          name: SetVisualStudioBaseBuildProductsDropName
+          displayName: Set 'VisualStudio.BaseBuild.ProductsDropName'
+          retryCountOnTaskFailure: 3
+          condition: ne(variables['VisualStudio.BaseBuild.ID'], '')
+          inputs:
+            filePath: $(DartLab.Path)\Scripts\VisualStudio\Build\Get-VisualStudioDropName.ps1
+            arguments: -AccessToken (ConvertTo-SecureString '$(System.AccessToken)' -AsPlainText -Force) -DropNamePrefix 'Products' -AccountURL '$(System.CollectionUri)' -ProjectName '$(System.TeamProject)' -BuildID $(VisualStudio.BaseBuild.ID) -OutVariableName 'VisualStudio.BaseBuild.ProductsDropName'
+
+        - task: PowerShell@2
+          name: SetBaseProductsDropNameToTarget
+          displayName: Set 'VisualStudio.BaseBuild.ProductsDropName' to drop from target build
+          retryCountOnTaskFailure: 3
+          condition: eq(variables['VisualStudio.BaseBuild.ID'], '')
+          inputs:
+            targetType: 'inline'
+            script: |
+              $buildDrop = "$(VsBaseBuildDrop)" -split "/"
+              $dropName = "Products/DevDiv/VS/$($buildDrop[-2])/$($buildDrop[-1])"
+              Write-Host "BaseBuildDrop: $buildDrop"
+              Write-Host "DropName: $dropName"
+              Write-Host "##vso[task.setvariable variable=VisualStudio.BaseBuild.ProductsDropName]$dropName"
+
+        - task: PowerShell@2
+          displayName: 'Set RunSettingsURI variable'
+          name: SetRunSettingsURI
+          inputs:
+            targetType: 'inline'
+            script: |
+              try {
+                $branchName = "$(resources.pipeline.ComponentBuildUnderTest.sourceBranch)"
+                $branchName = $branchName.Replace('refs/heads/', '')
+
+                $RunSettingsURI = "https://vsdrop.corp.microsoft.com/file/v1/RunSettings/$(System.TeamProject)/NuGet-NuGet.Client-Trusted/$branchName/$(resources.pipeline.ComponentBuildUnderTest.runID);NuGet.Tests.Apex.runsettings"
```

Comment:
We want to create a separate assembly for the daily tests, so we probably ened a separate run settings.

Reply:
Can we please create a follow up for this?

Reply:
Created a new _**NuGet.Tests.Apex.Daily.runsettings**_ for this pipeline.